runtime/src/accounts_db.rs

```diff
@@ -1121,6 +1121,29 @@ impl AccountsDB {
                 }
             }
         }
+
+        let mut counts = HashMap::new();
+        for slot_list in accounts_index.account_maps.values() {
```

Comment:
@ryoqun I think this should be more accurate to capture the updated count, can you take a look?

Reply:
Yeah, this looks good! This's kind of reverse of [my implementation](https://github.com/solana-labs/solana/pull/7013/files). Simper, nicer :)